packages/server/src/campaign/campaign.schema.ts

```diff
@@ -0,0 +1,66 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document, SchemaTypes } from 'mongoose';
+import { Organizer } from 'src/organizer/organizer.schema';
```

Comment:
@hadjirulricky let's update this to use the relative path instead of the path alias

`import { Organizer } from '../organizer/organizer.schema';`

This will throw an error when we transpile TypeScript files to JavaScript files